### PR TITLE
Start adding scholars, check pull request process

### PR DESCRIPTION
**Pull request by jcnorby**
Added a few scholars at the top of the list, and wanted to make sure this process worked for adding people to the repo.

### CODE REVIEW

File: _people/ChinweEkenna.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+---
+Prof. Ekenna directs the [Robotics Algorithm and Computable Systems (RACS) Laboratory](http://www.cs.albany.edu/RACS/index.html), which focuses on the development of intelligent systems, motion planning in complex spaces, heterogeneity analysis of the workspace, metabolic pathways, and protein folding.
+* C. Ekenna, S. A. Jacobs, S. Thomas and N. M. Amato, ["Adaptive neighbor connection for PRMs: A natural fit for heterogeneous environments and parallelism,"](https://ieeexplore.ieee.org/abstract/document/6696510) 2013 IEEE/RSJ International Conference on Intelligent Robots and Systems, Tokyo, 2013, pp. 1249-1256.
+* C. Ekenna and L. Tapia, ["Best Practices of Successful Academic Research Labs [Women in Engineering],"](https://ieeexplore.ieee.org/abstract/document/8930905) in IEEE Robotics & Automation Magazine, vol. 26, no. 4, pp. 138-141, Dec. 2019.
+* A. Upadhyay, W. Wang and C. Ekenna, ]"Approximating Cfree Space Topology by Constructing Vietoris-Rips Complex,"](https://ieeexplore.ieee.org/document/8968148) 2019 IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS), Macau, China, 2019, pp. 2517-2523.
```
Comment (aaronjoh) [inline _people/ChinweEkenna.md:12]: Bracket backwards on final paper link, inconsistent quotation mark use.

File: _people/ChrisCrawford.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Chris S.
+last_name: Crawford
+title: Assistant Professor of Computer Science
+school: University of Alabama
+website: http://www.chrissmithcrawford.com/
+image: https://cs.ua.edu/wp-content/uploads/2017/08/1708052_MW_049_chris_crawford-800x1000.jpg
+---
+Prof. Crawford directs the [Human-Technology Interaction Lab](https://htil.github.io/). His work focuses on Brain-Computer Interfaces (BCI) and Human-Robot Interaction (HRI).
+* Marvin Andujar, Chris S. Crawford, Anton Nijholt, France Jackson & Juan E. Gilbert (2015) [Artistic brain-computer interfaces: the expression and stimulation of the user’s affective state](https://www.tandfonline.com/doi/abs/10.1080/2326263X.2015.1104613), Brain-Computer Interfaces, 2:2-3, 60-69
```
Comment (aaronjoh) [inline _people/ChrisCrawford.md:10]: Date (and rest of citation) format in this first one does not match the others

File: _people/MuyinatuBell.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Muyinatu A.
+last_name: Lediju Bell
+title: Assistant Professor of Electrical and Computer Engineering,  Biomedical Engineering, and Computer Science
+school: Johns Hopkins University
+website: https://engineering.jhu.edu/ece/faculty/muyinatu-bell/
+image: https://api.hub.jhu.edu/factory/sites/default/files/styles/landscape/public/muyinatu_bell082316.jpg?itok=RfzKL2sI
+---
+Prof. Bell is a pioneer in the field of medical imaging technology. She founded and directs the [Photoacoustic and Ultrasonic Systems Engineering (PULSE) Lab](https://pulselab.jhu.edu/), an interdisciplinary research effort to develop novel ultrasound and photoacoustic imaging system components for diagnostic and surgical applications.
```
Comment (aaronjoh) [inline _people/MuyinatuBell.md:9]: Bio should highlight the robotics aspects of her reserch

Comment (jcnorby) [reply _people/MuyinatuBell.md:9]: I was just copying in entries from the doc to check that everything was working. Do we want to perfect each entry in the doc before we port them over, or just move them and revise here? Similarly, the doc has no standard citation format at the moment (part of why I was just suggesting links + a Bibtex). We could standardize if we want to go the way of formatted citations.

File: _people/MuyinatuBell.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Muyinatu A.
+last_name: Lediju Bell
+title: Assistant Professor of Electrical and Computer Engineering,  Biomedical Engineering, and Computer Science
+school: Johns Hopkins University
+website: https://engineering.jhu.edu/ece/faculty/muyinatu-bell/
+image: https://api.hub.jhu.edu/factory/sites/default/files/styles/landscape/public/muyinatu_bell082316.jpg?itok=RfzKL2sI
+---
+Prof. Bell founded and directs the [Photoacoustic and Ultrasonic Systems Engineering (PULSE) Lab](https://pulselab.jhu.edu/). Her lab introduced the concept of teleoperative photacoustic-guided surgery using the da Vinci surgical system, the first study of its kind combining photoacoustic imaging and minimally invasive robotic surgeries to improve accuracy. She pioneers work in medical imaging technology, robot-assisted imaging, andmachine learning for image formation.
```
Comment (aaronjoh) [inline _people/MuyinatuBell.md:9]: Typo "andmachine"

File: _people/MuyinatuBell.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Muyinatu A.
+last_name: Lediju Bell
+title: Assistant Professor of Electrical and Computer Engineering,  Biomedical Engineering, and Computer Science
+school: Johns Hopkins University
+website: https://engineering.jhu.edu/ece/faculty/muyinatu-bell/
+image: https://api.hub.jhu.edu/factory/sites/default/files/styles/landscape/public/muyinatu_bell082316.jpg?itok=RfzKL2sI
```
Comment (aaronjoh) [inline _people/MuyinatuBell.md:7]: Remove the itok part of this url

File: _people/ChrisCrawford.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+website: http://www.chrissmithcrawford.com/
+image: https://cs.ua.edu/wp-content/uploads/2017/08/1708052_MW_049_chris_crawford-800x1000.jpg
+---
+Prof. Crawford directs the [Human-Technology Interaction Lab](https://htil.github.io/). His work focuses on Brain-Computer Interfaces (BCI) and Human-Robot Interaction (HRI).
+* Andujar, Marvin, et al. "[Artistic brain-computer interfaces: the expression and stimulation of the user’s affective state.](https://www.tandfonline.com/doi/abs/10.1080/2326263X.2015.1104613)" Brain-computer interfaces 2.2-3 (2015): 60-69.
+* Crawford, Chris S., et al. "[User experience evaluation towards cooperative brain-robot interaction.](https://link.springer.com/chapter/10.1007/978-3-319-20901-2_17)" International Conference on Human-Computer Interaction. Springer, Cham, 2015.
+* Crawford, Chris S., Christina Gardner-McCune, and Juan E. Gilbert. "[Brain-computer interface for novice programmers.](https://dl.acm.org/doi/pdf/10.1145/3159450.3159603?casa_token=c0QZtkaVYO4AAAAA:HJw2viFZ0rK_2IrPk4EApLAY51hxQVni_yNrU2_eM9JV55dyVVuFdIhFUPSSpcTPRVyp51vIH3zn-g)" Proceedings of the 49th ACM Technical Symposium on Computer Science Education. 2018.
```
Comment (aaronjoh) [inline _people/ChrisCrawford.md:12]: Remove the casa_toke part of the url (everything starting with the ?

File: _people/ChinweEkenna.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+* Ekenna, Chinwe, et al. "[Adaptive neighbor connection for PRMs: A natural fit for heterogeneous environments and parallelism.](https://ieeexplore.ieee.org/abstract/document/6696510)" 2013 IEEE/RSJ International Conference on Intelligent Robots and Systems. IEEE, 2013.
+* Aakriti, Upadhyay, Wang Weifu, and Ekenna Chinwe. "[Approximating Cfree Space Topology by Constructing Vietoris-Rips Complex.](https://ieeexplore.ieee.org/document/8968148)" Proceedings of the International Conference on Intelligent Robots and Systems. 2019.
+* Tran, Tuan, and Chinwe Ekenna. "[Metabolic pathway and graph identification of new potential drug targets for Plasmodium Falciparum.](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=8217947&casa_token=4ipSbLxlatsAAAAA:rS7P9T3ilb_Ckty2Ifrm5w_jBGSd_B44XaCEDIdAso4ZmjmnPI0aR0jzFhikNNACeI2xAeqPrA)" 2017 IEEE International Conference on Bioinformatics and Biomedicine (BIBM). IEEE, 2017.
```
Comment (aaronjoh) [inline _people/ChinweEkenna.md:12]: Remove the casa_token part of the URL (everything after &)

Comment (jcnorby) [reply _people/ChinweEkenna.md]: Cleaned these up, take a look (I'll keep an eye out for more of those).

**Pull request merged**